include/fast_float/fast_table.h

```diff
@@ -36,662 +36,663 @@ constexpr static int smallest_power_of_five = binary_format<double>::smallest_po
 constexpr static int largest_power_of_five = binary_format<double>::largest_power_of_ten();
 constexpr static int number_of_entries = 2 * (largest_power_of_five - smallest_power_of_five + 1);
 // Powers of five from 5^-342 all the way to 5^308 rounded toward one.
-static const uint64_t power_of_five_128[number_of_entries];
```

Comment:
We need to make sure that the large table is included only once per project. I.e., if several compilation units include our header, we want the large table to appear only once in the final executable or library. That is why we use the strange template construction with an unused parameter.

Reply:
I am *not* objecting to the PR. I am just raising a flag.